docs/guide/logs-guide.md

```diff
@@ -1,33 +1,91 @@
+Collect and send data from any part of your stack with different log shippers available, and centralize and index logs from operating systems, containers, network devices, AWS infrastructure, application access logs, custom events, and more.
 
-Collect and send data from any part of your stack with different [log shippers available](https://sematext.com/docs/logs/sending-log-events/), and centralize and index logs from operating systems, containers, network devices, AWS infrastructure, application access logs, custom events, and more. 
+Register for free or Login into Sematext IT systems monitoring platform to get started and create your logs app. Upload your logs from all your servers to our centralized log management solution with Elasticsearch API and integrated Kibana, and experience the first true Hosted ELK Stack.
 
-Our [logging management platform](https://sematext.com/logsene/) provides ability to create your own queries through its [Elasticsearch API](https://sematext.com/docs/logs/search-through-the-elasticsearch-api/) as well as rich, yet simple query syntax very much like the [query syntax](https://sematext.com/docs/logs/search-syntax/) used by Google. Search and analyze your IT enterprise logs using Sematext advanced analytics tools, identify and compare patterns and anomalies, and monitor your whole IT infrastructure through a single pane of glass. 
+If you prefer good old way to search and analyze your logs, try our command line tool for viewing, searching, filtering, and analyzing of log entries across all your logs from all your apps.
 
-You can also create and integrate [alerts](https://sematext.com/docs/alerts/) with commonly used collaboration and messaging software such as [Hipchat](https://sematext.com/docs/integration/alerts-hipchat-integration/), [Slack](https://sematext.com/docs/integration/alerts-slack-integration/), [Big Panda](https://sematext.com/docs/integration/alerts-bigpanda-integration/), [PagerDuty](https://sematext.com/docs/integration/alerts-pagerduty-integration/), [VictorOps](https://sematext.com/docs/integration/alerts-victorops-integration/), [OpsGenie](https://sematext.com/docs/integration/alerts-opsgenie-integration/) or simply by using your email address(es). Use these services to continuously monitor large volume of data and logs, and ensure that you are alerted in time.
+## What is Sematext Logs?
+Think of Sematext Logs as logging as a service or your own logging box in the cloud (or on your own infrastructure if you really can't ship logs out to the cloud).
+
+It is cheaper alternative to Splunk, or even as Hosted Elasticsearch, since one of the APIs Logsene exposes is Elasticsearch API for indexing and searching.
```

Comment:
@adnanrahic  Do you want to link some of the things here to some things under https://sematext.com/resources/industry-comparisons/ ?
Also, "Logsene" in this context should just be "Sematext".

Reply:
Also note that "It is cheaper alternative to Splunk, or even as Hosted Elasticsearch" is a little off.

Reply:
I've fixed this in the `master` branch. It should be fine now.